lib/task/physicalobject/physicalObjectDeleteUnlinkedTask.class.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * Delete unlinked physical objects
+ *
+ * @package    symfony
+ * @subpackage task
+ * @author     Mike Cantelon <mike@artefactual.com>
+ */
+class physicalObjectDeleteUnlinkedTask extends arBaseTask
+{
+  protected $namespace        = 'physicalobject';
+  protected $name             = 'delete-unlinked';
+  protected $briefDescription = "Delete physical objects that aren't linked to descriptions";
+
+  protected $detailedDescription = <<<EOF
+Delete physical objects that aren't linked to descriptions
+EOF;
+
+  /**
+   * @see sfBaseTask
+   */
+  protected function configure()
+  {
+    $this->addOptions(array(
+      new sfCommandOption('application', null, sfCommandOption::PARAMETER_OPTIONAL, 'The application name', true),
+      new sfCommandOption('env', null, sfCommandOption::PARAMETER_REQUIRED, 'The environment', 'cli'),
+      new sfCommandOption('connection', null, sfCommandOption::PARAMETER_REQUIRED, 'The connection name', 'propel'),
+      new sfCommandOption('verbose', 'v', sfCommandOption::PARAMETER_NONE, "Verbose (shows details of what's marked for deletion", null),
+      new sfCommandOption('force', 'f', sfCommandOption::PARAMETER_NONE, 'Delete without confirmation', null),
+      new sfCommandOption('dry-run', 'd', sfCommandOption::PARAMETER_NONE, 'Dry run (no database changes)', null),
+    ));
+  }
+
+  /**
+   * @see sfTask
+   */
+  public function execute($arguments = array(), $options = array())
+  {
+    parent::execute($arguments, $options);
+
+    // Offer to abort if not using --force or --dry-run options
+    if (!$options['force'] && !$options['dry-run'])
+    {
+      $confirmation = $this->askConfirmation("Are you sure you'd like to delete all unlinked physical objects?");
+
+      if (!$confirmation)
+      {
+        $this->log('Aborted.');
+        exit();
+      }
+    }
+
+    // Disable search index
+    QubitSearch::disable();
+
+    // Display initial count of physical objects
+    $physicalObjectsCountBefore = $this->getPhysicalObjectCount();
+
+    $this->log();
```

Comment:
Is this to add an empty line in the output? Maybe adding `\n` to the following log looks a little better, but if it's not completely necessary I'd remove the empty line from the logs.